app/api/gltf/gltf.controller.js

```diff
@@ -84,13 +84,15 @@ async function getDbImage(image_id) {
 
   const result = await query
 
-  const searchImagePromise = [];
+  const iiifLevel0Promise = [];
 
-  if (result.dataValues.media && result.dataValues.media.image_url === null && result.dataValues.iiif_data) {
-    searchImagePromise.push(loadIIIFLevel0Utils.getUrlOnImage(result.dataValues.media, result.dataValues.iiif_data.size_info, 1024));
+  const media = result.media;
```

Comment:
There is a difference one time result.dataValues and in your code result.media.
Is it the fix you have done in gltf controller?

Reply:
yes, result.dataValues does not exist